templates/_helpers.tpl

```diff
@@ -106,7 +106,7 @@ Get the mastodon secret.
 {{- if .Values.mastodon.secrets.existingSecret }}
     {{- printf "%s" (tpl .Values.mastodon.secrets.existingSecret $) -}}
 {{- else -}}
-    {{- printf "%s" (include "common.names.fullname" .) -}}
+    {{- printf "%s" (include "mastodon.fullname" .) -}}
```

Comment:
This change does not seem to be part of the PR?

Reply:
Not sure what you mean by "not part of the PR".
But this is a change I made while working on this. `common.names.fullname` isn't a value that exists (found it out during testing) so I fixed it quickly.